### PR TITLE
Fix duplicate key warning and various other console warnings

### LINKED ISSUE #3046: Comments can be created twice with the same clientID. Duplicate key warning shows.
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
No duplicate keys occur for FlatList items in a chat

## Actual Result:
In some rare cases (probably related to the offline network queue stuff) comments get created twice and we end up with duplicate keys for `FlatList`.  While the duplicate comments are concerning having duplicate keys could be screwing things up and was first spotted by @parasharrajat though I'm not sure whether we got to the bottom of it.

## Workaround:
Yes, but not too sure whether there are bad effects.

## Platform:
Where is this issue occurring?

 Web
 iOS
 Android
 Desktop App
 Mobile Web

**Version Number:**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by marcaaron**
### Details
Just cleaning up a bunch of random warnings that are in the console. 

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/3046

### Tests
1. Booted iOS and Web apps then did various stuff. Didn't seem like there were any major issues.

### QA Steps
No QA

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [ ] Android

### CODE REVIEW

File: src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
```
Comment (deetergp) [inline src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js:1]: NAB: Just curious why you chose to declare the prop types in their own file.

**Review by deetergp — APPROVED** [review]
Death to console warnings! Code looks good to me.

File: src/pages/home/report/ReportActionsView.js [hunk 2]
```diff
@@ -391,7 +391,11 @@ class ReportActionsView extends React.Component {
                 renderItem={this.renderItem}
                 CellRendererComponent={this.renderCell}
                 contentContainerStyle={[styles.chatContentScrollView]}
-                keyExtractor={item => `${item.action.clientID}` || `${item.action.sequenceNumber}`}
+
+                // We use a combination of sequenceNumber and clientID in case the clientID are the same - which
+                // shouldn't happen, but might be possible in some rare cases.
+                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                keyExtractor={item => `${item.action.sequenceNumber}${item.action.clientID}`}
```
Comment (parasharrajat) [inline src/pages/home/report/ReportActionsView.js:398]: Dang. Nailed it.

File: src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js (hunk 1 above)
Comment (marcaaron) [reply src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js:1]: Just to DRY things up as they are the same in both native and web versions of the `PressableWithSecondaryInteraction`

**Pull request merged**